test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -1,7 +1,7 @@
 test_duration: 500
-n_db_nodes: 1
+n_db_nodes: 3
 n_loaders: 1
-n_monitor_nodes: 1
+n_monitor_nodes: 0
```

Comment:
BTW: why number of monitors is 0? @roydahan

Reply:
Don't know, I think it's better to have it 1.